config/crd/bases/anywhere.eks.amazonaws.com_cloudstackmachineconfigs.yaml

```diff
@@ -56,26 +56,20 @@ spec:
                   size including number of CPU’s, memory, and CPU speed. It can either
                   be specified as a UUID or name
                 properties:
-                  type:
+                  id:
```

Comment:
Is this the PR where we are refactoring the changes from `type`/`value` to `id`/`name`? If so, can we split the PR into logical chunks so that it is easier to review?